meson.build

```diff
@@ -476,6 +476,26 @@ if elogind.found()
   conf.set('HAVE_LOGIND' , '1')
 endif
 
+if get_option('launchd').disabled()
+  use_launchd = false
+else
+  launchctl = find_program('launchctl',
+    required: get_option('launchd').disable_auto_if(host_machine.system() != 'darwin'),
+  )
+  use_launchd = cc.has_header('launch.h') and launchctl.found()
+  if get_option('launchd').enabled() and not use_launchd
+    error('launchd support requested but not found')
+  endif
```

Comment:
this one can also go, as `launchctl = find_program()` has a `required:` check AFAICT